.github/workflows/ci.yml

```
@@ -83,7 +83,7 @@ jobs:
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - name: make
-        run: make -j3 SERVER_CFLAGS='-Werror'
+        run: make all-with-unit-tests -j3 SERVER_CFLAGS='-Werror'
```

make flags before the target seems more common style. We do that in other places.

```suggestion
        run: make -j3 all-with-unit-tests SERVER_CFLAGS='-Werror'
```